pkg/utils/configs/manifest.go

```diff
@@ -0,0 +1,208 @@
+package configs
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/skupperproject/skupper/pkg/images"
+	"os"
+	"strings"
+)
+
+type SkupperImage struct {
+	Name       string `yaml:"name"`
```

Comment:
should struct tag be json?

Reply:
It should definitely, thanks! 

Fixed in 258203f9f1cb0d93e4572d8d8ec1d5a87dc3230d